README.md

```diff
@@ -1,182 +1,177 @@
+<img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width=300px>
+
+## Node-wot
```

Comment:
```suggestion
## node-wot
```

lower-case?

Reply:
checked also playground is upper case. committing.